[2025-09-22T04:23:19.277+0000] {processor.py:186} INFO - Started process (PID=44) to work on /opt/airflow/dags/schedule.py
[2025-09-22T04:23:19.280+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/schedule.py for tasks to queue
[2025-09-22T04:23:19.291+0000] {logging_mixin.py:190} INFO - [2025-09-22T04:23:19.290+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/schedule.py
[2025-09-22T04:23:19.301+0000] {logging_mixin.py:190} INFO - [2025-09-22T04:23:19.300+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/schedule.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/schedule.py", line 5, in <module>
    import argpars
ModuleNotFoundError: No module named 'argpars'
[2025-09-22T04:23:19.302+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/schedule.py
[2025-09-22T04:23:19.324+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/schedule.py took 0.053 seconds
[2025-09-22T04:23:49.730+0000] {processor.py:186} INFO - Started process (PID=109) to work on /opt/airflow/dags/schedule.py
[2025-09-22T04:23:49.731+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/schedule.py for tasks to queue
[2025-09-22T04:23:49.734+0000] {logging_mixin.py:190} INFO - [2025-09-22T04:23:49.733+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/schedule.py
[2025-09-22T04:23:49.742+0000] {logging_mixin.py:190} INFO - [2025-09-22T04:23:49.739+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/schedule.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/schedule.py", line 5, in <module>
    import argpars
ModuleNotFoundError: No module named 'argpars'
[2025-09-22T04:23:49.743+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/schedule.py
[2025-09-22T04:23:49.765+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/schedule.py took 0.044 seconds
[2025-09-22T04:24:25.710+0000] {processor.py:186} INFO - Started process (PID=44) to work on /opt/airflow/dags/schedule.py
[2025-09-22T04:24:25.711+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/schedule.py for tasks to queue
[2025-09-22T04:24:25.719+0000] {logging_mixin.py:190} INFO - [2025-09-22T04:24:25.719+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/schedule.py
[2025-09-22T04:24:25.727+0000] {logging_mixin.py:190} INFO - [2025-09-22T04:24:25.726+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/schedule.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/schedule.py", line 5, in <module>
    import argpars
ModuleNotFoundError: No module named 'argpars'
[2025-09-22T04:24:25.728+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/schedule.py
[2025-09-22T04:24:25.744+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/schedule.py took 0.039 seconds
